stetho/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.facebook.stetho">
```

Comment:
You should add this: `xmlns:tools="http://schemas.android.com/tools"` here. Otherwise Travis CI will fail because the XML namespace could not be recognized by the parser.